Src/FluentAssertions/Types/MethodInfoSelector.cs

```diff
@@ -133,6 +133,24 @@ public MethodInfoSelector ThatAreNotDecoratedWithOrInherit<TAttribute>()
         return this;
     }
 
+    /// <summary>
+    /// Only return methods that are abstract
+    /// </summary>
+    public MethodInfoSelector ThatAreAbstract()
+    {
+        selectedMethods = selectedMethods.Where(method => method.IsAbstract);
```

Comment:
For consistency let's use `IsCSharpAbstract()` here as well.

Reply:
I couldn't come up with an example as well, but that does not imply that no such example exist, that's why I wanted to go for consistency.

But let's just go ahead and assume we're right until someone comes up with a counter example.